Source/WebCore/storage/StorageNamespaceProvider.h

```diff
@@ -46,15 +46,22 @@ class StorageNamespaceProvider : public RefCounted<StorageNamespaceProvider> {
     WEBCORE_EXPORT StorageNamespaceProvider();
     WEBCORE_EXPORT virtual ~StorageNamespaceProvider();
 
-    virtual Ref<StorageNamespace> createSessionStorageNamespace(Page&, unsigned quota) = 0;
-
     Ref<StorageArea> localStorageArea(Document&);
+    Ref<StorageArea> sessionStorageArea(Document&);
+
+    enum class ShouldCreateIfNotExists : bool { No, Yes };
```

Comment:
`ShouldCreateIfNotExists` sounds a little weird. I guess `ShouldCreateIfItDoesNotExist` would be less weird but long. Maybe `ShouldCreateIfNecessary`?